sbpy/spectroscopy/sun/core.py

```diff
@@ -9,8 +9,8 @@
 import astropy.units as u
```

Comment:
`astropy.units` is only used in the doctests currently and does not need to be imported here